### ISSUE TITLE
Bridge update

### ISSUE THREAD

**Issue by dirkson** [matrix-hacks/matrix-puppet-hangouts#13]
Heyo. The bridge version has changed under the project again. Would someone mind updating it?

Cheers!

**Comment by AndrewJDR** [comment]
Can you clarify what you mean?
I looked on npmjs and it appears to match the latest version on github...

**Comment by dirkson** [comment]
https://www.npmjs.com/search?q=matrix-appservice-bridge - Well, this page says that matrix-appservice-bridge is on version 1.4.x now. But when I type 'npm install' for matrix-puppet-hangouts, it immediately looks for 1.3.7, which doesn't exist.

Since one of the commit notes was "upgrade to latest bridge base (1.13.0)", I figured there might be some sort of version upgrade process needing done. But I don't know NPM or Javascript, so I figured it might be easier for everyone if I just asked.

Cheers!
